CHANGELOG.md

```diff
@@ -1,5 +1,97 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-rc.0
+
+<!-- generated comparing v5.0.0-beta.5..next -->
+
+_Sep 1, 2021_
+
+A big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 Renamed packages to `@mui/*`. This is our huge step to go beyond being material-design component library. More details about the [plan](https://github.com/mui-org/material-ui/issues/27825) and the [why](https://github.com/mui-org/material-ui/discussions/27803)
+- 🛠 Added `mui-replace` codemod for migrating `@material-ui/*` to new packages `@mui/*`. checkout this [codemod detail](https://github.com/mui-org/material-ui/blob/next/packages/material-ui-codemod/README.md#mui-replace) or head to [migration guide](https://next.material-ui.com/guides/migration-v4/#preset-safe)
+- 🧪 Added new `<Mansory>` component to the lab, [check it out](https://next.material-ui.com/components/masonry/). It has been crafted by our first intern! @hbjORbj 👏.
+
+### `@mui/material@5.0.0-rc.0`
+
+#### Breaking changes
+
+- &#8203;<!-- 32 -->[core] Rename packages (#28049) @mnajdova
+
+  The packages were renamed. Here is the mapping between the old and new names:
+
+  ```
+  @material-ui/core -> @mui/material
+  @material-ui/system -> @mui/system
+  @material-ui/unstyled -> @mui/core
+  @material-ui/styles -> @mui/styles
+  @material-ui/icons -> @mui/icons-material
+  @material-ui/lab -> @mui/lab
+  @material-ui/types -> @mui/types
+  @material-ui/styled-engine -> @mui/styled-engine
+  @material-ui/styled-engine-sc ->@mui/styled-engine-sc
+  @material-ui/private-theming -> @mui/private-theming
+  @material-ui/codemod -> @mui/codemod
+  @material-ui/docs -> @mui/docs
+  @material-ui/envinfo -> @mui/envinfo
+  ```
+
+  We encourage you to use the [codemod](https://github.com/mui-org/material-ui/blob/next/packages/material-ui-codemod/README.md#mui-replace) for smooth migration.
+
+#### Changes
+
+- &#8203;<!-- 38 -->[Autocomplete] Update warning for `value` prop (#27977) @vedadeepta
+- &#8203;<!-- 37 -->[Button] Create ButtonUnstyled and useButton (#27600) @michaldudak
```

Comment:
I guess that's more for the Core package (although Material's ButtonBase was also changed to use `useButton`, so it's hard to decide, really)

Reply:
I would add it to the ~`unstyled`~ (haha 😄) `@mui/core` package. Changing the `ButtonBase` to use the hook should be just an internal that does not have any effect on the developers using the library.

Reply:
Ah, I missed this. I think it should be in ###unstyled section.